sites/docs/src/content/docs/checklists/reviews/components_pr.md

```diff
@@ -35,5 +35,6 @@ Otherwise, you can cover most of the specifications by checking for the followin
 
 - [ ] Check that there are tests for all outputs, including optional ones.
 - [ ] Check that the `meta.yml` file has correct documentation links and patterns of files.
+- [ ] Check that the `meta.yml` file has correct `bio.tools` ID for the tool and correct EDAM ontology links of files.
```

Comment:
we need to link to the EDAM ontology whenever we mention it the first time in a file